aws/sagemaker-mxnet-inference-toolkit
# infra: Make docker folder read only, remove unused tests, update repo name, remove incorrect documentation.

* Make docker folder read only
* Remove unused tests
* Rename test-toolkit/ -> test/
* Rename buildspec.yml -> buildspec-container-pr.yml 
* Rename buildspec-toolkit.yml -> buildspec.yml 
* Update repo name
* Remove incorrect documentation.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
